keg_elements/db/mixins.py

```diff
@@ -295,7 +295,13 @@ def update_collection(self, attr_name, data):
 
 
 class DefaultMixin(DefaultColsMixin, MethodsMixin):
-    pass
+    @classmethod
```

Comment:
Sorry, I should have been more clear.  This logic should go in the already existing `MethodsMixin.random_data_for_column` instead of being part of a sub-class.